nengo_gui/conftest.py

```diff
@@ -0,0 +1,27 @@
+from selenium import webdriver
+import time
+import pytest
+import nengo_gui
+import os
+# each function should get the same driver...
+
+@pytest.fixture(scope="module")
+def driver(request):
+    # try:
+    #     folder = os.getcwd()
+    #     driver = webdriver.Chrome(os.path.join(folder,'chromedriver'))
+    # except:
+    driver = webdriver.Firefox()
+
+    driver.get('localhost:8080/')
+    driver.maximize_window()
+    time.sleep(4)
+    def fin():
+        driver.close()
+    request.addfinalizer(fin)
+    try:
+        assert driver.title != "Problem loading page"
+    except AssertionError as e:
+        print 'NENGO_GUI SERVER NOT STARTED, START SERVER BEFORE RUNNING TESTS!'
```

Comment:
I'd prefer to avoid the all-caps in a message like this.  How about:

"ERROR: The nengo_gui server is not currently running.  Start the server before running tests."